Runtime/AddressablePrefab.cs

```diff
@@ -73,7 +72,7 @@ static void SetRecursiveFlags(Transform transform)
         }
         static void DestroyChildren(Transform transform)
         {
-            for (int i = 0; i < transform.childCount;)
+            for (int i = 0; i < transform.childCount; i++)
                 DestroyImmediate(transform.GetChild(i).gameObject);
```

Comment:
This loop will now fail.
As transform children are destroyed, transform.childCount will reduce, but adding to i will move it up and this will leave some children undeleted in some circumstances.

Reply:
I take the blame for this, as it should have always been a while loop and this code is "clever" misuse of a for loop